exchanges/coinbene/coinbene.go

```diff
@@ -1189,7 +1191,10 @@ func (c *Coinbene) SendAuthHTTPRequest(ep exchange.URL, method, path, epPath str
 		Verbose:       c.Verbose,
 		HTTPDebugging: c.HTTPDebugging,
 		HTTPRecording: c.HTTPRecording,
-		Endpoint:      f,
+	}
+
+	if err := c.SendPayload(ctx, f, func() (*request.Item, error) {
```

Comment:
This function also uses timestamp based signature/headers. I would advise inlining. Coinbene has rate limits